### PR TITLE
Add merged cells metadata in tabular module

### PR DESCRIPTION
**Pull request by rodionnv**

### CODE REVIEW

File: s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    @OWLAnnotationProperty(iri = CSVW.nameUri)
+    private String name;
+
+    @OWLAnnotationProperty(iri = CSVW.RowUri)
+    private String rowName;
+
+    @OWLAnnotationProperty(iri = CSVW.ColumnUri)
+    private String columnName;
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java:31]: Does not make sense to have type:
csvw:Column

Why it uses columnName and not IRI ?

File: s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            this.setUri(new URI(cellUri));
+        } catch (URISyntaxException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @OWLAnnotationProperty(iri = CSVW.nameUri)
+    private String name;
+
+    @OWLAnnotationProperty(iri = CSVW.RowUri)
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java:27]: see comments for columnName

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+<http://test-file#row-4>
+        <http://onto.fel.cvut.cz/data/bb>
+                "ff" ;
+        <http://onto.fel.cvut.cz/data/cc>
+                "gg" .
+
+<http://test-file#cell2>
+        a       <http://www.w3.org/ns/csvw#Cell> ;
+        <http://www.w3.org/ns/csvw#Column>
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:38]: not correct http://www.w3.org/ns/csvw#Column is type not rdf property

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                _:b1 ;
+        <http://www.w3.org/ns/csvw#Row>
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:40]: csvw:Row is type not rdf property

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl
Comment (blcham) [inline s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:1]: I do not understand why it is called `merged-xls-model-output.ttl` ....  how about `xls-with-merged-cells-` ...

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl (hunk 3 above)
Comment (blcham) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:38]: BTW, is it actually working like that? So this is working test and you are saving blank-nodes? (as i know it is not that easy)

Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:38]: Yes, it is the actual output Model for the merged.xls

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl (hunk 4 above)
Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl]: Resolved in [this commit](https://github.com/kbss-cvut/s-pipes/pull/222/commits/670025b432666bdea700ca6d9cbb1e0e6ec3aff2).

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl (hunk 3 above)
Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl]: Wrong URI was resolved in [this commit](https://github.com/kbss-cvut/s-pipes/pull/222/commits/670025b432666bdea700ca6d9cbb1e0e6ec3aff2).

File: s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java (hunk 1 above)
Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java]: Resolved in [this commit](https://github.com/kbss-cvut/s-pipes/pull/222/commits/670025b432666bdea700ca6d9cbb1e0e6ec3aff2).

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl
Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl:1]: Resolved

File: s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl (hunk 4 above)
Comment (blcham) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl]: I looked at [CSVW specification](https://www.w3.org/ns/csvw#column) but it is not consistent with:

Column 
    - An array property of column descriptions as described in section 5.6 
    - rdfs:range csvw:Column
    - rdfs:domaincsvw:Schema


Thus statement,
?x a csvw:Cell .
?x csvw:column ?y .
-----------------------------
implies
----------------------------
?x a csvw:Schema 


And thus ?x is both csvw:Cell and csvw:Schema which we do not want.


Thus I suggest to use rather something like kbss-csvw:**is-part-of-column**/references-column/column ... 
I guess same applies for csvw:row

Comment (rodionnv) [reply s-pipes-modules/module-tabular/src/test/resources/merged-xls-model-output.ttl]: Resolved, with csvw:row it was a similar problem so now kbss-csvw:is-part-of-column and kbss-csvw:is-part-of-row are used instead

File: s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        }
+    }
+
+    @OWLAnnotationProperty(iri = CSVW.nameUri)
+    private String name;
+
+    @OWLAnnotationProperty(iri = KBSS_CSVW.isPartOfRow)
+    private String row;
+
+    @OWLAnnotationProperty(iri = KBSS_CSVW.isPartOfColumn)
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java:30]: I do not understand how returning String value can work

File: s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    private String column;
```
Comment (blcham) [inline s-pipes-modules/module-tabular/src/main/java/cz/cvut/spipes/modules/model/Cell.java:31]: I do not understand how returning String value can work when you are referencing blank node in data

**Pull request closed without merging**